src/components/pages/LockScreen.tsx

```diff
@@ -53,10 +53,46 @@ export const LockScreen = (): JSX.Element => {
   const biometryAvailable = Boolean(flagshipMetadata.biometry_available)
   const biometryType = flagshipMetadata.biometry_type
 
-  const onBiometryLock = (): void =>
-    void handleChange(setBiometry, 'biometryLock', webviewIntent).then(
-      value => value && setAutoLock(true)
-    )
+  const onBiometryLock = (): void => {
```

Comment:
we can't use an async method here?

Reply:
No according to React we have to provide a synchronous function returning nothing. Though it looks like a typescript issue in React more than a runtime issue

Reply:
I'm converting it to async due to a recent commit from Yannick, it will make the code more readable